### DISCUSSION TITLE
How to implement the release v5.77.0 with CRA

### DISCUSSION THREAD

**Discussion by rupeesdev** [webpack/webpack#16901] [Q&A] [answered]
Hi there! First of all, Im new at software development and GitHub, so please, go easy on me :). Im developing a migration of a React architecture from v16 to v18, and I'm experimenting the same error of previous discussions here. This architecture used `worker-loader` to solve the problem of importing the workers, but in webpack 5, this option is deprecated as you know. The thing is, I have a worker builder `new Worker(new URL( WebWorker, import.meta.url))` where, in the var WebWorker, I pass the URL of the worker that I want to import. This, of course, when I try to deploy in localhost it says the error that is mentioned in #14680, for example. I have tried the solution that the user @piotr-oles proposes in that very thread, but it didnt work for me (`error: Not allowed to load local resource: file:///C:/rute/of/the/worker/WorkerName.worker.js`)

I've seen that this particular issue is solved in the new release v5.77.0 with the PR #16671 but I'm using CRA and Im not 100% sure how to implement this change in my code, as long as I only use the webpack.config.js. Again, Im new using webpack and this is probably the most stupid question that you will ever read, but could you help me with this topic? Thanks you all for your help!

**Comment by alexander-akait** [comment, marked as answer]
Sorry, you need to report this to CRA repo, but you can inject your configuration and use any options, I think it will be good idea

**Comment by rupeesdev** [comment]
Okay, I will try, thank you!
